src/components/index.d.ts

```diff
@@ -890,6 +890,18 @@ export declare const Header: {
   }
 }
 
+interface NoraButtonProps {
+  className: string
+  children: React.ReactNode
+}
+export declare const NoraButton: {
```

Comment:
I'm pretty sure we'll need to expose some events on these buttons at minimum `onClick`

Reply:
oh true 🤦‍♀ will do that now